.travis.yml

```diff
@@ -5,15 +5,26 @@ env:
 install:
   - pip install GitPython
   - export TRAVIS_COMMIT_MSG="$TRAVIS_REPO_SLUG - $(git log --format=%B --no-merges -n 1)"
-  - git clone https://github.com/xbmc-catchuptv-au/repo-devel.git .deploy
 script:
-  - cd .deploy
-  - ./manage_repo.py ../
+  - 'if [ -z "$TRAVIS_TAG" ]; then
+         git clone https://github.com/glennguy/repo-devel.git .deploy || travis_terminate 1;
```

Comment:
I guess this shouldn't be your personal github account? :)

Reply:
heh no :)